source/features/pull-request-hotkey.tsx

```diff
@@ -4,19 +4,20 @@ import * as pageDetect from 'github-url-detection';
 import features from '.';
 
 function init(): void {
-	const tabs = select.all('.tabnav-tabs .tabnav-tab');
-	const selectedIndex = tabs.indexOf(select('.tabnav-tabs .selected')!);
+	const tabs = select.all('.tabnav-tabs a.tabnav-tab');
```

Comment:
`a` excludes some buttons which also have `.tabnav-tab` for some reason

Reply:
Worth adding a comment explaining what these are

Reply:
Replaced it with a better and self-explanatory selector.